### PR TITLE
JAVA-1540: DefaultVertex `toString()` prints `DefaultElement`.	

### PR DESCRIPTION
**Pull request by newkek**
Also wondering if it is necessary to print the "**Default**...." part of the element's kind, so I suggest we could just print the element's kind without the "Default" part.

### CODE REVIEW

**Comment by tolbertam** [comment]
I think it would be good to just remove `DefaultElement.toString()` altogether since it is abstract.  This way if we do add another implementation it will be more clear it needs it's own `toString()`

File: driver-core/src/main/java/com/datastax/driver/dse/graph/DefaultElement.java [hunk 1]
```diff
@@ -94,11 +94,10 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return "DefaultElement{" +
+        return "Element{" +
```
Comment (adutra) [inline driver-core/src/main/java/com/datastax/driver/dse/graph/DefaultElement.java]: I agree with @tolbertam I think we should get rid of this `toString()` method.

**Comment by newkek** [comment]
@adutra I think the comments you made have been processed, would you 👍 the PR?

**Comment by newkek** [comment]
_note (to self): merge this only after #76 is merged because `PrintElementsTest` will need to be fixed again_

**Pull request merged**